test/elixir_analyzer/test_suite/high_school_sweetheart_test.exs

```diff
@@ -0,0 +1,116 @@
+defmodule ElixirAnalyzer.TestSuite.HighSchoolSweetheartTest do
+  alias ElixirAnalyzer.Constants
+
+  use ElixirAnalyzer.ExerciseTestCase,
+    exercise_test_module: ElixirAnalyzer.TestSuite.HighSchoolSweetheart
+
+  test_exercise_analysis "example solution",
+    comments: [] do
+    defmodule HighSchoolSweetheart do
+      def first_letter(name) do
+        name
+        |> String.trim()
+        |> String.first()
+      end
+
+      def initial(name) do
+        name
+        |> first_letter()
+        |> String.upcase()
+        |> Kernel.<>(".")
+      end
+
+      def initials(full_name) do
+        [first_name, last_name] = String.split(full_name)
+        "#{initial(first_name)} #{initial(last_name)}"
+      end
+
+      def pair(full_name1, full_name2) do
+        i1 = initials(full_name1)
+        i2 = initials(full_name2)
+
+        """
+              ******       ******
+            **      **   **      **
+          **         ** **         **
+        **            *            **
+        **                         **
+        **     #{i1}  +  #{i2}     **
+          **                       **
+            **                   **
+              **               **
+                **           **
+                  **       **
+                    **   **
+                      ***
+                      *
+        """
+      end
+    end
+  end
+
+  describe "function reuse" do
+    test_exercise_analysis "HighSchoolSweetheart.initial must call first_letter",
+      comments_include: [Constants.high_school_sweetheart_function_reuse()] do
+      defmodule HighSchoolSweetheart do
```

Comment:
If you expect the same comments for a bunch of test modules, you can put them together in a list
```elixir
test_exercise_analysis "something",
  comments_include: [Constants.high_school_sweetheart_function_reuse()] do
  [ defmodule HighSchoolSweetheart do
     ...
     end,
     defmodule HighSchoolSweetheart do
     ...
     end
  ]
```

Reply:
That being said, if we split the comment in three, it can stay this way :)